onnx/defs/math/defs.cc

```diff
@@ -2006,6 +2007,273 @@ ONNX_OPERATOR_SET_SCHEMA(
           defs::math::utils::MatMulShapeInference(ctx, 0, 1);
         }));
 
+static const char* MatMulNBits_ver23_doc = R"DOC(
+MatMulNBits is a MatMul with weight quantized with N bits (e.g., 2, 3, 4, 5, 6, 7). It does Matrix Multiplication like
+[MatMul](#matmul) with differences:
+
+1. Input `B` is a 2D constant Matrix. Its input feature count and output feature count are specified by attributes 'K'
+    and 'N'.
+2. Input `B` is quantized with x bits which is specified by attribute 'bits'. It is quantized blockwisely along
+    dimension 0 (e.g. column) with block size specified by attribute block_size. And block_size is not an arbitrary
+    number and must be a power of 2 and not smaller than 16, like 16, 32, 64, 128,..
+3. Input `B`'s scale and zero point are specified by input scales and zero_points.
+
+    Input `B` is stored as uint8_t with shape: `[N][n_blocks_per_col][blob_size]` or
+    `[N][n_blocks_per_col * blob_size]`
+
+    in which:
+      - `n_blocks_per_col` = `(K + block_size - 1) / block_size`
+      - `blob_size` = `CeilDiv(block_size * bits, bitsof(uint8_t)<8>)`
```

Comment:
Can we replace `bitsof(uint8_t)<8>` by `8`? I am not sure I understand what is currently written.

Reply:
Yes, I can replace this with 8